src/sagemaker/remote_function/client.py

```diff
@@ -82,49 +82,174 @@ def remote(
+          also activate a new persistent cache feature, which will further reduce job start up
+          latency than over using SageMaker managed warm pools alone by caching the package source
+          downloaded in the previous runs.
+
+        max_retry_attempts (int): The max number of times the job is retried after an on
+          ```InternalServerFailure``` Error. Defaults to 1.
+
+        max_runtime_in_seconds (int): The upper limit in seconds to be used for training. After
+          this specified amount of time, SageMaker terminates the job regardless of its current
+          status. The max_run_duration time defaults to 1 day or (86400 seconds).
+
+        role (str): The IAM role (either name or full ARN) used to run your SageMaker training
+          job. Defaults to:
+
+          * the SageMaker default IAM role if the SDK is running in SageMaker Notebooks or
+            SageMaker Studio Notebooks.
+          * if not above, a ValueError is be thrown.
+
+        s3_kms_key (str): The key used to encrypt the input and output data. Default to ``None``.
+
+        s3_root_uri (str): The root S3 folder to which where the code archives and data are
+          uploaded to. Defaults to ``s3://<sagemaker-default-bucket>``.
+
+        sagemaker_session (sagemaker.session.Session): The underlying SageMaker session to which
+          SageMaker service calls are delegated to (default: None). If not provided, one is created
+          using a default configuration chain.
+
+        security_group_ids (List[str): A list of security group IDs. Defaults to ``None`` and the
+          training job is created without VPC config.
+
+        subnets (List[str): A list of subnet IDs. Defaults to ``None`` and the job is created
+          without VPC config.
+
+        tags (List[Tuple[str, str]): A list of tags attached to the job. Defaults to ``None`` and
+          the training job is created without tags.
+
+        volume_kms_key (str): An Amazon Key Management Service (KMS) key used to encrypt an
+          Amazon Elastic Block Storage (EBS) volume attached to the training instance. Defaults to
+          ``None``.
+
+        volume_size (int): The size in GB of the storage volume for storing input and output data
+          during training. Defaults to ``30``.
+
+        encrypt_inter_container_traffic (bool): A flag that specifies whether traffic between
+          training containers is encrypted for the training job. Defaults to ``False``.
+
+        enable_network_isolation (bool): A flag that specifies whether container will run in
+          network isolation mode. Defaults to ``False``. Network isolation mode restricts the
+          container access to outside networks (such as the Internet). The container does not
+          make any inbound or outbound network calls. Also known as Internet-free mode.
     """
 
     def _remote(func):
```

Comment:
all the comments as previous `def remote` function might apply here, please verify

Reply:
+1. Done.